From 1d307bec63fa2a94e0d32f89e01d7798a66f0910 Mon Sep 17 00:00:00 2001
From: Limeng <Meng.Li@windriver.com>
Date: Fri, 26 Oct 2018 15:17:42 +0800
Subject: [PATCH] arch: arm64: dts: add property snps,multicast-filter-bins
 for stratix10 platform

The property value of snps,multicast-filter-bins is used to calculate
hash table size of stratix10 platform via log2 algorithm.
Because there are 8 hash table registers, the property value is 256

Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/altera/socfpga_stratix10_socdk.dts    |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/altera/socfpga_stratix10_socdk.dts b/arch/arm64/boot/dts/altera/socfpga_stratix10_socdk.dts
index 745e78c..ae99c65 100644
--- a/arch/arm64/boot/dts/altera/socfpga_stratix10_socdk.dts
+++ b/arch/arm64/boot/dts/altera/socfpga_stratix10_socdk.dts
@@ -112,6 +112,7 @@
 	rxdv-skew-ps = <420>; /* 0ps */
 	rxc-skew-ps = <1680>; /* 780ps */
 	max-frame-size = <3800>;
+	snps,multicast-filter-bins = <256>;
 	status = "okay";
 };
 
-- 
1.7.9.5

